integration.js

```diff
@@ -33,15 +32,15 @@ const doLookup = async (entities, { url, ..._options }, cb) => {
       Logger
     );
   } catch (error) {
-    Logger.error(error, 'Get Lookup Results Failed');
+    Logger.error(error, `Get Lookup Results Failed ${Object.keys(entities)}`);
```

Comment:
`Object.keys(entities)` 
With `entities` being an array, the keys that will be logged will be the indices.  I'm not sure how useful this we be to clients or us when debugging, so I think this probably should be removed from this log.